### DISCUSSION TITLE
Unable to sign in / authentication not working

### DISCUSSION THREAD

**Discussion by knambiar** [requarks/wiki#3353] [Help / Questions]
**Describe the bug**
All users are unable to login to the wiki with the error "Invalid authentication provider". Could be caused after I tried to enable google sign in.

**To Reproduce**
1. Go to https://wiki.1stmain.co
2. Type in any username and password
3. Click on login
4. Error should now appear

**Expected behavior**
User should be logged in (or show an invalid password error)

**Screenshots**
![image](https://user-images.githubusercontent.com/29317917/71743152-ccd30480-2e89-11ea-86f7-ad38fc2d7704.png)

**Host Info (please complete the following information):**
 - OS: Docker on Ubuntu 18.04, Docker
 - Wiki.js version: 2.0.x
 - Database engine: postgres 9.7 (on docker)

**Comment by NGPixel** [comment]
Do you have any errors when restarting the app?

**Comment by knambiar** [comment]
Seems to start up ok. I also registered as a new user and signed in using the link generated to the email and that worked fine. 

I did also see an error message prior to this when creating a new page. Something along the lines of access error to /homes/user even though when i refreshed the page it was created fine.

**Comment by NGPixel** [comment]
Can you try to disable the google provider?

**Comment by knambiar** [comment]
Hi, I managed to do this by editing the db entry directly. I'm in now, however, every time I save the a page, I still get this error:

`EACEESS: permission denied, mkdir '/home/userid'`
(where userid is the user ID used on my server)

**Comment by knambiar** [comment]
A few more errors I'm seeing, when I try to create a new page, and sometimes when I'm loading an existing page I get this error:
net::ERR_INCOMPLETE_CHUNKED_ENCODING 200 (OK)

and I see a blank page.

**Comment by NGPixel** [comment]
Make sure the user running node has the proper write access rights to the wiki directory.

**Comment by knambiar** [comment]
I'm actually running the wikijs process inside a docker container. The docker container itself is running with the sudo command. Are you saying I need to modify user permissions inside the docker container?

**Comment by Trojaner** [comment]
I am facing the same issue after enabling keycloak login.
Disabling it will allow me to log in via local authentication again after next restart.
No matter what I try I can not get keycloak logins to work.

**Comment by doubleailes** [comment]
Same case for me here. Since activation google auth, my admin user doesn't work anymore.

**Comment by NGPixel** [comment]
Upgrade to 2.4.105

**Comment by Trojaner** [comment]
I still have this issue with Keycloak with latest helm installation
